lib/plug/static.ex

```diff
@@ -7,12 +7,12 @@ defmodule Plug.Static do
     * `:at` - the request path to reach for static assets.
       It must be a string.
 
-    * `:from` - the filesystem path to read static assets from.
-      It must be a string, containing a file system path, or an
-      atom representing the application name where assets will
-      be served from the priv/static, or a tuple containing the
-      application name and the directory to serve assets besides
-      priv/static.
+    * `:from` - the file system path to read static assets from.
```

Comment:
Also, since this is not really a "path" (can be an application, and so on), what if we say

> `:from` - the location where to read static assets from. It can be either ...